tests/Resources/testhost-custom-task.yml

```diff
@@ -14,7 +14,9 @@ magephp:
                 - composer/install
```

Comment:
Sorry but this test is nonsense. It increases coverage but it does not test anything at all. You might as well just drop it. Please add a test that checks if `CustomTask` was really registered :)